mutant/modules/sarscov2_delivery.py

```diff
@@ -61,14 +63,28 @@ def rename_deliverables(self):
                 newpath = "{0}/{1}.vcf".format(prefix, base_sample)
                 os.symlink(item, newpath)
 
-            # rename core
-            core_suffix = [
-                ".qc.csv",
-                ".typing_summary.csv",
-                ".variant_summary.csv",
-            ]
-            for thing in core_suffix:
-                hit = glob.glob("{0}/*{1}".format(self.indir, thing))
-                if len(hit) == 1:
-                    hit = hit[0]
-                    os.symlink(hit, "{0}/{1}{2}".format(self.indir, self.ticket, thing))
+
+        ## Rename case files
+
+        # rename multiqc
+        hit = glob.glob("{}/multiqc/*_multiqc.html".format(self.indir))
+        if len(hit) == 1:
```

Comment:
can just work with `if hit` cause [] == None

Reply:
Nah cause we want to make sure there is just a single file.